Content.Shared/Paint/PaintComponent.cs

```diff
@@ -0,0 +1,55 @@
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.FixedPoint;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Audio;
+using Content.Shared.Decals;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared.Paint;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+[Access(typeof(SharedPaintSystem))]
+public sealed partial class PaintComponent : Component
+{
+
+    /// <summary>
+    /// Noise made when paint applied.
+    /// </summary>
+    [DataField]
+    public SoundSpecifier Spray = new SoundPathSpecifier("/Audio/Effects/spray2.ogg");
+
+    /// <summary>
+    /// Solution on the entity that contains the paint.
+    /// </summary>
+    [DataField]
+    public string Solution = "drink";
+
+    [DataField, AutoNetworkedField]
+    public bool Painter = true;
+
+    /// <summary>
+    /// Reagent that will be used as paint.
+    /// </summary>
+    [DataField(customTypeSerializer: typeof(PrototypeIdSerializer<ReagentPrototype>))]
+    public string Reagent = "SpaceGlue";
```

Comment:
use 
`public ProtoId<ReafentPrototype> Reagent = "SpaceGlue"`

and remove
`using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;`

Reply:
changed this. Also thanks for this amazing review.